modules/serverless-rn-1-2-0.adoc

```diff
@@ -0,0 +1,45 @@
+// Module included in the following assemblies:
+//
+// * serverless/release-notes.adoc
+
+[id="serverless-rn-1-2-0_{context}"]
+//update the <version> to match the filename
+
+= Release Notes for Red Hat {ServerlessProductName} Technology Preview 1.2.0
+// add a version, e.g. Technology Preview 1.0.0
+
+[id="new-features-1-2-0_{context}"]
+== New features
+* {ServerlessProductName} has been updated to use Knative Serving 0.9.0.
+* {ServerlessProductName} has been updated to use Knative Client (`kn` CLI) 0.9.0.
+* {ServerlessProductName} on {product-title} 4.2 now uses the Operator Lifecycle Manager dependency resolution mechanism to install the ServiceMesh operator automatically. The required ServiceMeshControlPlane and ServiceMeshMemberRoll are also installed and managed for the user.
```

Comment:
Since the OLM acronym is used later in the module, suggest:

s/Operator Lifecycle Manager/Operator Lifecycle Manager (OLM)

for this first instance.